lib/Elastica/Query/Match.php

```diff
@@ -1,216 +0,0 @@
-<?php
-
-namespace Elastica\Query;
-
-/**
- * Match query.
- *
- * @author F21
- * @author WONG Wing Lun <luiges90@gmail.com>
- *
- * @deprecated since version 6.1.2, use the MatchQuery class instead.
- * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html
- */
-class Match extends AbstractQuery
```

Comment:
This should become `class Match extends MatchQuery` and otherwise be empty of content

Reply:
Keeping the comments above too, of course.